tests/charm++/Makefile

```diff
@@ -1,17 +1,11 @@
 DIRS = \
-  megatest \
```

Comment:
Changes to to this file seem unintentional

Reply:
yeah none of the makefile changes should be in there.  those bled over from a different test branch entirely.